src/data/registry.xml

```diff
@@ -315,10 +337,11 @@
       <data>reciprocal_of_dimensionless_exner_function_wrt_surface_air_pressure</data>
       <data>frontogenesis_function</data>
       <data>frontogenesis_angle</data>
-      <data>vertically_integrated_total_energy_of_initial_state_using_dycore_energy_formula</data>
-      <data>vertically_integrated_total_energy_of_current_state_using_dycore_energy_formula</data>
-      <data>vertically_integrated_water_vapor_and_condensed_water_of_initial_state</data>
-      <data>vertically_integrated_water_vapor_and_condensed_water_of_current_state</data>
+      <data>vertically_integrated_total_energy_using_dycore_energy_formula_at_start_of_physics_timestep</data>
+      <data>vertically_integrated_total_energy_using_dycore_energy_formula</data>
```

Comment:
Great question, I'm not sure! Would this only be necessary if it has to be accessed from SIMA code? I looked around in CAM and couldn't see a situation where it would need to, everything is contained within the `check_energy` scheme now.